pkg/util/glog/glog.go

```diff
@@ -67,7 +67,7 @@ func (gverbose) V(level int) Logger {
 }
 
 func (g gverbose) Infof(format string, args ...interface{}) {
-	g.Verbose.Infof(format, args...)
+	g.Verbose.InfoDepth(3, fmt.Sprintf(format, args...))
```

Comment:
ensures we print the actual line of the code that called the glog wrapper.

otherwise we get "glog:50" for every logged line.